src/View/MainMenu.fxml

```diff
@@ -149,6 +149,21 @@
                             <Button onAction="#importFile" style="-fx-background-color: transparent;-fx-text-fill:white; -fx-cursor: hand;"
                                     text="Import HUB file"
                             />
+                        </HBox>
```

Comment:
This looks like it doesn't belong here.  Looking at the file in its entirety, this creates an extra `</HBox>` in the GridPane that does not have a corresponding opening tag.  This will not render correctly.  Did you edit this FXML by hand or did you use a GUI designer?

Reply:
I looked at the syntax above and took what was used for say the hub file button and used the same syntax and everything and may have accidently added a extra HBox